qiskit/qasm/node/format.py

```diff
@@ -37,7 +37,7 @@ def qasm(self, prec=None):
         """Return the corresponding format string."""
         if prec is not None:
             warnings.warn(
-                "Parameter 'Format.qasm(..., prec)' is no longer used and is being " "deprecated.",
+                "Parameter 'Format.qasm(..., prec)' is no longer used and is being deprecated.",
```

Comment:
All of these strings in the QASM 2 module (and possibly some other modules in this PR) were already removed in #7208 - you probably just want to fix the merge conflicts by throwing out these changes.

Alternatively, if you did this PR by `sed` or similar, then maybe just rebase onto `main` and redo it - might be quicker!